### PR TITLE
Create views

### PR DESCRIPTION
**Pull request by Gambit142**
In this branch, I implemented the design from this [sneak peek wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,74 @@
...
+post2 = user1.posts.create!(title: 'Science', text: lorem_mauris, likes_counter: 0, comments_counter: 0)
+
+post3 = user2.posts.create!(title: 'Business', text: lorem_donec, likes_counter: 0, comments_counter: 0)
+post4 = user2.posts.create!(title: 'Education', text: lorem_mauris, likes_counter: 0, comments_counter: 0)
+
+post5 = user3.posts.create!(title: 'Sports', text: lorem_donec, likes_counter: 0, comments_counter: 0)
+post6 = user3.posts.create!(title: 'Agriculture', text: lorem_mauris, likes_counter: 0, comments_counter: 0)
+
+post7 = user4.posts.create!(title: 'Arts', text: lorem_donec, likes_counter: 0, comments_counter: 0)
+post8 = user4.posts.create!(title: 'Infrastructure', text: lorem_mauris, likes_counter: 0, comments_counter: 0)
```
Comment (AshakaE) [inline db/seeds.rb:23]: - [ ] Please fix the `likes_counter` typo here in the seeds and across your project, check the column name in your schema

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Gambit142  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/seeds.rb (hunk 1 above)
Comment (Gambit142) [reply db/seeds.rb:23]: Hi @AshakaE 👋, 

I am sorry but I think you got it wrong. I used `likes_counter` as a column for my database table. If you check the column table for posts in the `schema.rb`,  you will see this:

image

The `likes_counter` functions as you can see in the image below:
image

Finally, if the `likes_counter` was not functioning, the `create!` method will throw an error when creating the posts. 😃 👍

Comment (Dipeshtwis) [reply db/seeds.rb:23]: - Everything looks fine to me.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @Gambit142, 

Good job. Everything looks fine to me. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gambit142** [comment]
Thank you for your wonderful feedback

**Pull request merged**